tools/dependencies.props

```diff
@@ -3,10 +3,11 @@
     <RuntimeFrameworkVersion>2.1.0-preview2-26220-02</RuntimeFrameworkVersion>
     <CoreFxStableVersion>4.3.0</CoreFxStableVersion>
     <RoslynVersion>2.6.0-beta3-62316-02</RoslynVersion>
-    <SystemMemoryVersion>4.5.0-preview2-26221-01</SystemMemoryVersion>
-    <SystemCompilerServicesUnsafeVersion>4.5.0-preview2-26221-01</SystemCompilerServicesUnsafeVersion>
-    <SystemNumericsVectorsVersion>4.5.0-preview2-26221-01</SystemNumericsVectorsVersion>
-    <SystemBuffersVersion>4.5.0-preview2-26221-01</SystemBuffersVersion>
+    <SystemMemoryVersion>4.5.0-preview2-26221-05</SystemMemoryVersion>
+    <SystemCompilerServicesUnsafeVersion>4.5.0-preview2-26221-05</SystemCompilerServicesUnsafeVersion>
+    <SystemNumericsVectorsVersion>4.5.0-preview2-26221-05</SystemNumericsVectorsVersion>
+    <SystemBuffersVersion>4.5.0-preview2-26221-05</SystemBuffersVersion>
+    <SystemIOPipelinesVersion>4.5.0-preview2-26221-05</SystemIOPipelinesVersion>
```

Comment:
Can you update the program that auto-updates the versions to include this? See https://github.com/dotnet/corefxlab/blob/master/scripts/update-dependencies/Program.cs#L98